style.css

```diff
@@ -1,5 +1,1007 @@
+    box-sizing: border-box;
+    padding: 103px 24px 8px 24px;
+    border-radius: 76px 0 0 0;
+    background-image: url("images/fotb.png");
+    background-repeat: no-repeat;
+    background-position: right;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    width: 99vmax;
+    height: 846px;
+    align-items: center;
+    margin: 0 auto;
+    text-align: center;
+    justify-content: center;
+  }
+
+  .first-title-3 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+
+    /* or 130% */
+    height: 56px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    margin: 0;
+
+    /* Neutrals/N0 */
+
+    color: #fff;
+  }
+
+  .description-4 {
+    font-family: "poppins", sans-serif;
+    width: 569px;
+    height: 102px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+  }
+
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+  }
+
+  input {
+    background: #fff;
+    width: 447px;
+    height: 48px;
+    margin: 10px 0 10px 0;
+    padding-left: 20px;
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 17px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+  }
+
+  textarea {
+    background: #fff;
+    width: 447px;
+    height: 170px;
+    padding-left: 20px;
+    padding-top: 10px;
+    margin-top: 10px;
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 24px;
+    color: #b3bac5;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+  }
+
+  .highlight-4 {
+    background: #fff;
+    border-radius: 8px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    border: 1px solid white;
+    width: 150px;
+    margin-top: 15px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .font-6 {
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .highlight-4:hover {
+    background: #6070ff;
+    border-radius: 8px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    border: 1px solid #6070ff;
+    width: 150px;
+    margin-top: 15px;
+  }
+
+  .font-6:hover {
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: white;
+  }
```

Comment:
- [ ] Nice job styling the form button 👍  but I noticed that when I hovered over the button, it seemed to disappear leaving only its text. I suggest adding a white border to it on hover so that the button doesn't give off the impression of disappearing on hover.
-
![c2](https://user-images.githubusercontent.com/85113567/149405767-2c37301c-2663-4d61-87bf-b9fce3bf0742.jpg)

Reply:
Addressed 🔥